weave/trace/settings.py

```diff
@@ -0,0 +1,72 @@
+import os
+from contextvars import ContextVar
+from typing import Any, Optional, Union
+
+from pydantic import BaseModel, ConfigDict, PrivateAttr
+
+disabled = ContextVar("disabled", default=False)
```

Comment:
curious why you chose to use multiple ContextVars as opposed to 1 `settings` that has multiple properties